force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -241,8 +309,12 @@ public with sharing class FlowControlLoopsCollections {
         Set<String> stringSet = new Set<String>();
 
         //add strings to the set
+        for(String str : inputList){
+            stringSet.add(str);
+        }
+       
 
-        return null; // Replace null with the variable you used to store the result
+        return stringSet; // Replace null with the variable you used to store the result
```

Comment:
You can pass a List directly into the Set constructor and solve this with no iteration at all. 

```
 public static List<Integer> createAndPopulateList() {
      return new List<Integer>{1, 2, 3};
    }
 ```